### PR TITLE
web/satellite: update satellite emails

### PR DESCRIPTION
**Pull request by boshevski**
New designs for all of the satellite email templates.

- Including logo (optimized), other branding design updates (styling, colors, etc.), footer with links (support, terms, and social links).
- Tested in various devices for responsiveness and accessibility. 
- Using native system fonts for faster rendering and compatibility. 
- Including rel="noopener noreferrer" in all links. 

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

**Review by mobyvb — COMMENTED** [review]
Emails I tested in storj-up (they lgtm besides my nit comment):

- Welcome.html (activate with link)
- UnknownReset.html (forgot password nonexistent account)
- Forgot.html (forgot password existing user)
- PasswordChanged.html (reset password via authenticated session in UI)
- NewUserInvite.html (invite non-existing user)
- UnverifiedUserInvite.html (invite existing user who is unverified)
- ExistingUserInvite.html (invite existing verified user)
- EmailAddressVerification.html (first step project deletion or account deletion)
- RequestAccountDeletionSuccess.html (complete account deletion)
- LoginLockAccount.html (repeatedly attempt login with incorrect password)
- EmailChangeSuccess.html (successful email change)
- ActivationLockAccount.html (repeated bad submission of activation code)
- AccountAlreadyExists.html (create account with already verified email)

I haven't tested these:

- BillingFreezeNotification.html
- BillingWarning.html
- TrialExpirationEscalationReminder.html
- TrialExpirationReminder.html
- TrialExpired.html

Invite.html doesn't seem to be used, but we should confirm and remove it if so.

About to work on fixing the breaking tests

File: web/satellite/static/emails/AccountAlreadyExists.html [hunk 1]
```diff
@@ -1,599 +1,174 @@
...
-                        <div class="column"
-                            style="text-align: left;color: #8e959c;font-size: 14px;line-height: 21px;
-                            font-family: Helvetica, sans-serif; font-weight: 300;max-width: 600px;min-width: 320px;width: calc(28000% - 167400px);">
-                            <div style="margin: 12px 20px;">
-                                <div style="mso-line-height-rule: exactly;mso-text-raise: 4px;">
-                                    <p class="size-20"
-                                        style="Margin-top: 5px;Margin-bottom: 0;
-                                        font-family: Helvetica, sans-serif; font-weight: 300;font-size: 17px;
-                                        line-height: 26px;" lang="x-size-20">
-                                        <span>If this wasn't you, we recommend <a href="{{ .ResetPasswordLink }}" target="_blank" rel="noopener noreferrer">resetting your password</a>.
```
Comment (mobyvb) [inline web/satellite/static/emails/AccountAlreadyExists.html:529]: reset password link was removed in this file, causing two tests to fail. We either need to add the reset password link back to the email, or remove the checks for that link from the tests

Comment (boshevski) [reply web/satellite/static/emails/AccountAlreadyExists.html:529]: thanks, i just updated the template

File: web/satellite/static/emails/WelcomeWithCode.html [hunk 2]
```diff
@@ -1,310 +1,187 @@
...
+                     </tr>
+                     <tr>
+                      <td align="left" class="es-text-4388" style="padding:0;Margin:0;padding-bottom:10px;padding-top:10px"><p class="es-text-mobile-size-16" style="Margin:0;mso-line-height-rule:exactly;font-family:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';line-height:24px;letter-spacing:0;color:#111;font-size:16px">You're receiving this email because you signed up for a STORJ account. Confirm your email address by entering the code below to get started on the distributed cloud. Do not share this code with anyone.</p></td>
```
[the remark this answers is not in the document]
Comment (boshevski) [reply web/satellite/static/emails/WelcomeWithCode.html]: updated

File: web/satellite/static/emails/AccountAlreadyExists.html [hunk 3]
```diff
@@ -1,599 +1,177 @@
...
-                                            If this activity looks suspicious to you, notify our <a href="https://supportdcs.storj.io/hc/en-us" target="_blank" rel="noopener noreferrer">Support team</a>.
-                                            <br/><br/>
-                                            If your intention was to create an account on a different satellite, please select the appropriate satellite abbreviation from the drop down at
-                                                the top right of the screen when <a href="{{ .CreateAccountLink }}" target="_blank" rel="noopener noreferrer">creating an account</a>.
```
Comment (mobyvb) [inline web/satellite/static/emails/AccountAlreadyExists.html:533]: Looks like the removed `{{ .CreateAccountLink }}` is also causing tests to fail

Comment (boshevski) [reply web/satellite/static/emails/AccountAlreadyExists.html:533]: yeah i think we don't need that part

**Review by mobyvb — APPROVED** [review]

**Review by andriikotko — APPROVED** [review]

**Pull request merged**